voice-assistant/esp32-s3-box-lite.yaml

```
@@ -500,6 +500,7 @@ display:
     dc_pin: 4
     reset_pin: 48
     update_interval: never
+    invert_colors: false
```

Please don't add changes to a PR if you have not tested it
```suggestion
    invert_colors: true
```